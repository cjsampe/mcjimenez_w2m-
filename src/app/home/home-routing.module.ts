import { NgModule } from '@angular/core';
import { RouterModule, Routes } from '@angular/router';
import { HomeComponent } from './home.component';



const route: Routes = [
  {
    path: '',
    component: HomeComponent,
  },
];

@NgModule({
  declarations: [],
  imports: [RouterModule.forChild(route)],
  exports: [RouterModule],
})

export class HomeRoutingModule { }
