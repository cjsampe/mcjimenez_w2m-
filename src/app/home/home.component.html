<!-- <img src="assets/img/funko_1.jpeg" alt="superheroes"> -->
<img src="assets/img/funko_3.jpg" alt="superheroes">
<h3>{{ 'HOME.TEXT' | transloco }}</h3>
<img src="assets/img/funko_2.jpg" alt="superheroes">

