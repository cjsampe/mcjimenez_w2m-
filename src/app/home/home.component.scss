img{
    margin: 2em, 2em, 2em,2em;  
    float: center; 
    width: 100%;
}
h1{
    text-align: center;
}