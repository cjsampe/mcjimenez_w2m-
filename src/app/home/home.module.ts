import { NgModule } from '@angular/core';
import { CommonModule } from '@angular/common';
import { HomeComponent } from './home.component';
import { TranslocoModule } from '@ngneat/transloco';



@NgModule({
  declarations: [
    HomeComponent
  ],
  imports: [
    CommonModule,
    TranslocoModule
  ],
  exports: [
  ]
})
export class HomeModule { }
