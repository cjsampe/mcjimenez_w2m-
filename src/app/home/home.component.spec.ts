import { ComponentFixture, TestBed } from '@angular/core/testing';
import { TranslocoModule } from '@ngneat/transloco';

import { HomeComponent } from './home.component';

describe('HomeComponent', () => {
  let component: HomeComponent;
  let fixture: ComponentFixture<HomeComponent>;
  let compiled: HTMLElement;

  beforeEach(async () => {
    await TestBed.configureTestingModule({
      imports:[TranslocoModule],
      declarations: [HomeComponent],
    }).compileComponents();

    fixture = TestBed.createComponent(HomeComponent);
    component = fixture.componentInstance;
    fixture.detectChanges();
    compiled= fixture.nativeElement;
  });

    //create component
  it('should create', () => {
    expect(component).toBeTruthy();
  });

    //snapshot
  it('must be equal to snapshot', () => {
    expect(compiled).toMatchSnapshot();
  });

});
