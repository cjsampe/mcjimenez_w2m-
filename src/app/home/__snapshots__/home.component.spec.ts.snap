// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeComponent must be equal to snapshot 1`] = `
<div
  id="root1"
>
  <img
    alt="superheroes"
    src="assets/img/funko_3.jpg"
  />
  <h3>
    en.HOME.TEXT
  </h3>
  <img
    alt="superheroes2"
    src="assets/img/funko_2.jpg"
  />
</div>
`;
